core/src/test/scala/other/kafka/TestLinearWriteSpeed.scala

```diff
@@ -101,14 +108,21 @@ object TestLinearWriteSpeed {
     val messageSize = options.valueOf(messageSizeOpt).intValue
     val flushInterval = options.valueOf(flushIntervalOpt).longValue
     val compressionCodec = CompressionCodec.getCompressionCodec(options.valueOf(compressionCodecOpt))
+    val extraProps = CommandLineUtils.parseKeyValueArgs(options.valuesOf(propertyOpt).asScala)
     val rand = new Random
     rand.nextBytes(buffer.array)
     val numMessages = bufferSize / (messageSize + Records.LOG_OVERHEAD)
     val createTime = System.currentTimeMillis
+    val properties = new Properties()
+    properties ++= extraProps
+    properties.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:9000")
```

Comment:
Ah right this is only created in order to call `getCompressionConfig`. For a second I was wondering where `localhost:9000` came from.